dev/sample_mqttplugin/__init__.py

```diff
@@ -78,6 +82,10 @@ def run(self):
 
         self.alive = True
 
+        # let the plugin change the state of pause_item
+        if self._pause_item:
```

Comment:
pause_item or pause_item_path..?

Reply:
pause_item, as the item needs to be accessible for it to be changed in the next line.